libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -4,40 +4,76 @@
       autoFocus
       matInput
       type="search"
-      placeholder="Search for books to add to your reading list"
+      placeholder="{{ constants.CONSTANTS.SEARCH_BOOKS }}"
       formControlName="term"
     />
-    <button mat-icon-button matSuffix>
-      <mat-icon>search</mat-icon>
+    <button
+      mat-icon-button
+      matSuffix
+      aria-label="Search button"
+      [disabled]="!searchForm.valid"
+      data-testing="search-button"
+    >
+      <mat-icon aria-label="Search icon">{{
+        constants.CONSTANTS.SEARCH
+      }}</mat-icon></button
+    ><button
+      mat-icon-button
+      matSuffix
+      aria-label="Reset Search"
+      data-testing="clear-button"
+      (click)="resetSearch()"
+    >
+      <mat-icon>{{ constants.CONSTANTS.CLEAR }}</mat-icon>
     </button>
   </mat-form-field>
 </form>
 
-<ng-container *ngIf="searchTerm; else empty">
+<ng-container *ngIf="(books$ | async).length; else empty">
   <div class="book-grid">
-    <div class="book" data-testing="book-item" *ngFor="let b of books">
-      <div class="book--title">
+    <div class="book" data-testing="book-item" *ngFor="let b of books$ | async">
+      <div class="book-title">
         {{ b.title }}
       </div>
-      <div class="book--content">
-        <div class="book--content--cover">
-          <img src="{{ b.coverUrl }}" />
+      <div class="book-content">
+        <div class="book-content-cover">
+          <img src="{{ b.coverUrl }}" alt="Book image" title="{{ b.title }}" />
         </div>
-        <div class="book--content--info">
-          <div><strong>Author:</strong> {{ b.authors.join(',') }}</div>
-          <div><strong>Publisher:</strong> {{ b.publisher }}</div>
+        <div class="book-content-info">
           <div>
-            <strong>Published:</strong> {{ formatDate(b.publishedDate) }}
+            <strong>{{ constants.CONSTANTS.AUTHOR }}:</strong>
+            {{
+              b.authors.join(',') ? b.authors.join(',') : 'No Author mentioned'
```

Comment:
Added ternary operator which displays a proper message if the data is not available.
```suggestion
              b.authors.join(',') ? b.authors.join(',') : 'No Author mentioned'
```